adapters/postgres/postgres_test.go

```diff
@@ -791,3 +791,24 @@ func TestColumnsByRequest(t *testing.T) {
 		}
 	}
 }
+
+func TestTimeout(t *testing.T) {
```

Comment:
This test is a connection test.. the correct file to write this test is the `conn_test.go` in connection package